course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- Authorship on data, authorship in collaboration
+    - Now that you're a little deeper into the course, you're ready for a more in-depth discussion about scientific authorship. The readings here are meant to present some extreme views and really provoke discussion. Please read these readings prior to class on Thursday.
+- Visualization for communication
+    - For this discussion, there's a lot of material we could potentially look at, but the idea is to give you a taste of the things we need to consider when thinking about our visualizations. Conceptually, this is pretty open ended, but I want you to leave with an idea of why they should put a little bit more thought into graphing their data.
+- Academic publishing
+    - So, how do you decide where to publish? It’s easy to find resources about choosing a journal from a more conventional framework (TL:DR- most of them are some variation of “Choose the journal with the highest impact factor that also fits your work”)- but issues around open access muddies this old advice a bit. This is a really good opportunity to talk about issues with valuing science and scientists by impact factors of journals. We suggest reading High-impact open access journals and Eigenfactor Index of Open Access Fees to get you started. Open access seems simple on the surface, but we get into a very tangled web of corporate interests, biased metrics, and misinformation very quickly.
+- Participatory models for bigger science
+    - With increased connectivity, we have the opportunity to bring our work to audiences that have never been engaged in science before. This is a good opportunity for students to explore the differences in participatory models between fields, and discuss what engagement means in different fields. We provide a more extreme example, with participatory biomedical research- how do we engage people in our work when the direct ties to their well-being are more tenuous? Next week we'll look at participatory models in ecology.
+- Check up on the collaborative process
+    - An informal discussion that doubles as a chance to flesh out where the project is going. How is the collaborative process working? Do we need to make adjustments to get the project done? Use our answers to these questions to help populate/adjust the github issue list.
+- Looking towards the future
+    - Use this discussion as an opportunity to get feedback on the course, and to help guide students in embracing open science and reproducible research in their future endeavours. 
+
+
+### Exercises
+These are largely covered by the topics in the lectures, but if you see something that is not and that you think is important, comment on that line suggesting where to fit it in.
+
+- Find university IP policy, and discuss
+- Sign up for github
```

Comment:
this can be part of a "getting ready for the course" package that we make which includes installation information for R, RStudio, necessary packages, github account and git. This should be given to students prior to course starting but should be done by the latest by the end the lab of the first week.